packages/expo/src/modules/register/registerFirstStep/styles.ts

```diff
@@ -0,0 +1,162 @@
+import styled, {css} from 'styled-components/native';
+import {px2ddp} from '@animavita/theme';
+import {LinearGradient} from 'expo-linear-gradient';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+
+interface TypeProps {
+  isAnimalChecked: boolean;
+  onPress?: (event: Event) => void;
+}
+
+export const TypeButtons = styled(LinearGradient)<TypeProps>`
+  ${props =>
+    props.isAnimalChecked
+      ? css`
+          align-items: center;
+          padding-top: 4px;
+          padding-bottom: 4px;
+          padding-left: 4px;
+          padding-right: 4px;
```

Comment:
We could have just a `padding: 4px;` here (the same occurs more times in this same file)